lib/chai/utils/proxify.js

```diff
@@ -26,8 +26,10 @@ module.exports = function proxify (obj) {
     get: function getProperty (target, property) {
       // Don't throw error on Symbol properties such as Symbol.toStringTag, nor
       // on .then because it's necessary for promise type-checking.
```

Comment:
Since the `.then` default exclusion is no longer defined here, I think it'd be best to move the explanation on line 28 to the config file. I think the note you added on lines 29-30 is enough.